qiskit/_quantumprogram.py

```diff
@@ -268,7 +273,12 @@ def create_classical_register(self, name=None, size=1):
             return self.__classical_registers[name]
 
         if name is None:
-            name = self._create_id('c', self.__classical_registers)
+            name = self._create_id('c%i', self.__classical_registers)
```

Comment:
Can you check if this is correct? The call to `_create_id` accepts a string (without format-spec identifiers) as a first parameter, and this might end up producing names such as `c%i9`.

Reply:
ups.. fixed in cbc4b4e